ui/components/DashboardComponent/index.js

```diff
@@ -108,12 +109,69 @@ const styles = (theme) => ({
 
 const DashboardComponent = (props) => {
   const { classes, k8sconfig, selectedK8sContexts, updateProgress } = props;
+  const { notify } = useNotification();
 
   const [tabVal, setTabVal] = React.useState(0);
 
   const handleChange = () => (event, newValue) => {
     setTabVal(newValue);
   };
+  const fetchData = async () => {
```

Comment:
Please define the purpose.

Reply:
after formalizing the correct Rest Api design I"ll define the purpose

Reply:
When we load the site due to useEffect on page load it pings the ```api/meshmodels/registrants``` first to get the registrants which were registered and then by using its hostname we are passing to the ```api/meshmodels/nonRegisterEntity``` that returns the event of success and error both from the endpoint